File: DataAcquisition.Core/Models/DTOs/FacilityDto.cs
using System.Collections.Generic;
using DataAcquisition.Core.Models.Entities;

namespace DataAcquisition.Core.Models.DTOs
{
    public class FacilityDto
    {
        public int FacilityId { get; set; }
        public string FacilityName { get; set; }
        public string Address { get; set; }
        public int Employees { get; set; }
        public virtual ICollection<Workstation> Workstations { get; set; }
    }
}

File: DataAcquisition.Web/acquisition-web-app/src/app/models/facilityDto.ts
import { IWorkstationName } from '../interfaces/workstation-name';

export class FacilityDto {

    constructor(
      public facilityId = 0,
      public facilityName = '',
      public address?: string,
      public employees = 0,
      public workstations: IWorkstationName[] = []) { }
  }
